File: JavaScript/LinearSearch.js
const prompt = require('prompt-sync')({sigint:true});
let n = prompt("Enter no of elements: ");
let arr = [];
console.log("Enter elements : ---| ");
for(i=0;i<n;i++){
    let a = prompt();
    arr.push(a); 
}
const x = prompt("Enter value to be found: ");
let found = false;
for(i=0;i<n;i++){
    if(x === arr[i]){
        console.log("Found at index "+i+" !!");
        found = true;
        break;
    }
}
if(!found){
    console.log("Element Not Found !!");
}

File: C language/Searches/LinearSearch.c
#include<stdio.h>

int main(void){
    int arr[100];
    int n;
    int i;
    int x;
    int found = 0;
    printf("Enter N.O elements: ");
    scanf("%d",&n);
    printf("Enter elements:  ");
    for(i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    printf("Input has Taken ...\n");
    printf("Enter element to be searched: ");
    scanf("%d",&x);
    for(i=0;i<n;i++){
        if(arr[i]==x){
            printf("Element is found at index %d !!",i);
            found = 1;
            break;
        }
    }
    if(!found){
        printf("Element not Found !!");
    }
}